roles/manifest/README.md

```diff
@@ -0,0 +1,69 @@
+theforeman.foreman.manifest
+===============================================
```

Comment:
Can we use the same markdown formatting at the top level README or is this style something common for Ansible roles?

Reply:
It's a common format that is created when doing `ansible-galaxy init rolename`:

```
$ ansible-galaxy init test-role-1
- Role test-role-1 was created successfully
$ cat test-role-1/README.md
```
Which creates the skeleton:

``` 
Role Name
=========

A brief description of the role goes here.

Requirements
------------

Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.

Role Variables
--------------

A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role. Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.

Dependencies
------------

A list of other roles hosted on Galaxy should go here, plus any details in regards to parameters that may need to be set for other roles, or variables that are used from other roles.

Example Playbook
----------------

Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:

    - hosts: servers
      roles:
         - { role: username.rolename, x: 42 }

License
-------

BSD

Author Information
------------------

An optional section for the role authors to include contact information, or a website (HTML is not allowed).
```

Reply:
Great, thank you for sharing that.

Reply:
FWIW, I don't think we need to adhere to that template by the word. Like license and author seems overly duplicated with other metadata we already have.